script.js

```diff
@@ -0,0 +1,70 @@
+const button = document.getElementById('btn');
+const body = document.body;
+const menuField = document.querySelector('.wrap');
+const gameField = document.createElement('div');
+body.appendChild(gameField);
+let isCardClick = false;
+
+const addCardBox = () => {
+  const cardBox = document.createElement('div');
+  gameField.appendChild(cardBox);
+  cardBox.className = 'card_box';
+
+  const shirt = document.createElement('div');
+  shirt.innerHTML="<img src='images/card.png'>";
+  cardBox.appendChild(shirt);
+  shirt.className = 'shirt';
+
+  const faceCard = document.createElement('div');
+  faceCard.className = 'face';
+  let rand = Math.floor(Math.random() * 2);
+  if (rand === 0) {
+    faceCard.innerHTML = "<img src='images/cardBug.png'>";
+  } else if (rand === 1) {
+    faceCard.innerHTML = "<img src='images/cardTheEnd.png'>";
+  }
+  cardBox.appendChild(faceCard);
+
+  const cardOnClick = () => cardBox.classList.toggle('flip');
+  cardBox.addEventListener('click', () => {
+    if (isCardClick) {
+      gameField.style.display = 'none';
+      menuField.style.display = '';
+      isCardClick = false;
+    } else {
+    	cardOnClick();
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)